client/dashboard/sites/site/index.tsx

```diff
@@ -44,7 +45,7 @@ function Site() {
 							renderContent={ ( { onClose } ) => <Switcher onClose={ onClose } /> }
 						/>
 					</HeaderBar.Title>
-					{ canViewStagingSite( site ) && (
+					{ ! hasSiteTrialEnded( site ) && canViewStagingSite( site ) && (
```

Comment:
Shouldn't `canViewStagingSite()` return `false` for expired trials?

Reply:
I initially did that, but I'm not sure they should be tied together. The staging site might still be accessible even after the trial plan expires. In this case, we just want to hide the switcher if the trial has expired, so the purpose seems slightly different. What do you think?